indico/modules/events/editing/operations.py

```diff
@@ -158,7 +158,7 @@ def review_editable_revision(revision, editor, action, comment, tags, files=None
                                        tags=tags)
         revision.editable.revisions.append(new_revision)
     db.session.flush()
-    notify_editor_judgment(revision, editor)
+    notify_editor_judgment(new_revision, revision.user, revision_type)
```

Comment:
I think passing the `action` here would be better - for instance, when doing an `update_accept`, it will show up in the email as just an acceptance, which is wrong